base/version.jl

```diff
@@ -208,6 +208,13 @@ catch e
     global const VERSION = VersionNumber(0)
 end
 
+"""
+    VERSION
+
+An object describing which version of Julia is in use.
```

Comment:
further down it looks like you were just porting over the existing rst description, so consider this an enhancement :)